recipes/keystone/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES C)
+
+find_package(keystone REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.c)
```

```suggestion
project(test_package LANGUAGES CXX)

find_package(keystone REQUIRED CONFIG)

add_executable(${PROJECT_NAME} test_package.cpp)
```

I would suggest promoting test package to C++ package, as the project is C++ too.